src/draw-page.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useRef } from "react";
+import { useTranslation } from "react-i18next";
+import DrawingWidget, {
+  ImageData,
+} from "./components/drawing-widget/drawing-widget.component";
+import { createAttachment } from "./attachments/attachments.resource";
+import { useParams } from 'react-router-dom';
+import html2canvas from "html2canvas";
+
+const DrawPage: React.FC = () => {
+
+  const { patientUuid } = useParams();
+  useTranslation();
+  const [activeImage] = useState<ImageData | null>(null);
```

Comment:
If you're using `useState` without obtaining and using both the state variable and the setter, you're doing something very very wrong.